modules/ximgproc/samples/find_ellipses.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python

```

Comment:
EOL (end-of-line) encoding of this file is CRLF (Windows).
Please switch to LF (UNIX style)

Some hints are here: https://stackoverflow.com/questions/27810758/how-to-replace-crlf-with-lf-in-a-single-file